resources/PythonPip3Dsc/PythonPip3Dsc.psm1

```diff
@@ -4,6 +4,77 @@
 using namespace System.Collections.Generic
 
 #region Functions
+function Invoke-Process
+{
+    [CmdletBinding()]
+    param
+    (
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [string]$FilePath,
+
+        [Parameter()]
+        [ValidateNotNullOrEmpty()]
+        [string]$ArgumentList,
+
+        [ValidateSet("Full", "StdOut", "StdErr", "ExitCode", "None")]
+        [string]$DisplayLevel
+    )
+
+    try
+    {
+        $pinfo = New-Object System.Diagnostics.ProcessStartInfo
+        $pinfo.FileName = $FilePath
+        $pinfo.RedirectStandardError = $true
+        $pinfo.RedirectStandardOutput = $true
+        $pinfo.UseShellExecute = $false
+        $pinfo.WindowStyle = 'Hidden'
+        $pinfo.CreateNoWindow = $true
+        $pinfo.Arguments = $ArgumentList
+        $p = New-Object System.Diagnostics.Process
+        $p.StartInfo = $pinfo
+        $p.Start() | Out-Null
+
+        $stOut = @()
+        while (-not $p.StandardOutput.EndOfStream)
+        {
+            $stOut += $p.StandardOutput.ReadLine()
+        }
```

Comment:
I think you can do $stdError = $p.StandardOutput.ReadToEnd(). Same for StandardError